builder/googlecompute/step_create_instance.go

```diff
@@ -24,13 +24,17 @@ func (c *Config) createInstanceMetadata(sourceImage *Image, sshPublicKey string)
 		instanceMetadata[k] = v
 	}
 
-	// Merge any existing ssh keys with our public key.
-	sshMetaKey := "sshKeys"
```

Comment:
still trying to understand what this section is doing.

It looks like you can specify the authorized keys on the instance when it runs by setting the Metadata? 

So if you're manually setting the private key, you must already be setting the public key in the authorized keys file. is that correct?

Would we maybe want to get the public key from the private key file and automatically add it to the metadata so users don't have to do that manually? Or is that not a common use-case. I'm not a gce user.